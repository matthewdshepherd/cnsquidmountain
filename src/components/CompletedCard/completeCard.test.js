import React from 'react'
import { shallow } from 'enzyme'
import CompletedWarmUpCard from './completedCard'

describe('completeCard', () => {
  it('Should match the snapshot with all data passed in correctly', () => {
    const prefix = { prefix: 'e/ex', id: 1 }
    const wrapper = shallow(<CompletedWarmUpCard prefix={prefix} meaning={prefix} />)
    expect(wrapper).toMatchSnapshot()
  })
})
