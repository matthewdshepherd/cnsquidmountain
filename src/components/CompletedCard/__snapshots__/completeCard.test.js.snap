// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`completeCard Should match the snapshot with all data passed in correctly 1`] = `
<div
  className="completedWord__container"
  key="1"
>
  <div
    className="compltedword--word--definition"
  >
    <div
      className="compltedword--word--pos"
    >
      <p
        className="completedWord__p--word"
      >
        e/ex
      </p>
    </div>
  </div>
  <p
    className="completedWord__p--pos"
  >
    e/ex
  </p>
</div>
`;
